roles/installer/tasks/main.yml

```diff
@@ -1,11 +1,10 @@
 ---
-- name: Check for presence of Deployment
+- name: Check for presence of web deployment
   k8s_info:
     api_version: v1
     kind: Deployment
-    name: "{{ ansible_operator_meta.name }}"
+    name: "{{ ansible_operator_meta.name-web }}"
```

Comment:
variable ansible_operator_meta.name-web is not a exist 

ansible_operator_meta.name exist 

so this line should be `name: "{{ ansible_operator_meta.name }}-web"`

it appends -web to whatever `ansible_operator_meta.name` resolve to